cmd/main.go

```diff
@@ -24,6 +24,7 @@ import (
 	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
 	kustomizecommonv1 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/common/kustomize/v1"
+	configgenv1alpha "sigs.k8s.io/kubebuilder/v3/pkg/plugins/config-gen/v1alpha"
```

Comment:
Hi @Kavinjsir, 

The icon in the title needs to be ✨  (we are adding a new plugin), and I would recommend something like
> `:sparkles`: New plugin for config-gen as alpha. 

Also, could you ensure that all commits of this PR are squashed

Reply:
Sounds good!